assets/js/go-back.js

```diff
@@ -3,12 +3,20 @@ document.addEventListener(
   function () {
     var goBackLinks = document.querySelectorAll(".go-back-link");
 
+    function hasUsableHistory() {
+      return window.history.length > 1 && document.referrer !== "";
```

Comment:
Question : why do you need to test document.referrer?

Reply:
If I don't add it, we can go back to the search engine. That's exactly what we're trying to avoid.
I think it also blocks feedback on FS.